src/main.go

```diff
@@ -1,22 +1,56 @@
 package main
 
-import  (
+import (
 	"flag"
-	"github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/SpecReader"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/Fetcher"
 	html "github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/HTML"
+	"github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/SpecReader"
 )
 
 func main() {
+	var repos []Fetcher.Repo
+
+	fetch := flag.String("fetch", "", "This command will fetch files from a list of git repos.\n \nHere is an example\n \n go run main.go --fetch \"https://github.com/Agile-Arch-Angels/decentralized-tech-radar_dev main ./Fetcher/something.txt https://github.com/JonasSkjodt/CopenhagenBuzz master ./Fetcher/something.txt https://gitlab.com/nagyv-gitlab/gitops-test master ./Fetcher/something.txt\"")
 	file := flag.String("file", "", "This takes a path to a csv file/string")
 	flag.Parse()
 
-	var specs SpecReader.Blips
-	// testing csv reader
-	if *file != "" {
-		specs = SpecReader.ReadCsvSpec(*file)
-	} else {
-		panic("No file was given (oh no)")
+	if *fetch != "" {
+		fetchArgs := strings.Split(*fetch, " ")
+		if len(fetchArgs) < 3 {
```

Comment:
Sure it could be used instead but I don't see how that is better than just doing < 3 for the purposes that we are using it for 🤷 Feel free to elaborate if you have a reason for changing it

Reply:
It can, yes, but either is fine here for what this specific code is designed for.

Reply:
Well if 4 arguments are given than it would be the wrong amount of arguments